js/main.js

```diff
@@ -0,0 +1,63 @@
+// функция для генерации случайного числа в заданном диапазоне
+function getRandomInt(a, b) {
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+}
+
+// массив с набором комментариев
+const messages = [
```

Comment:
Константы должны быть в верхнем регистре
```suggestion
const MESSAGES = [
```